tensorflow_cloud/python/tests/integration/on_script_call_test.py

```diff
@@ -0,0 +1,72 @@
+# Copyright 2020 Google LLC. All Rights Reserved.
```

Comment:
Can we rename file to run_on_script_test.py?

Reply:
done